blog/2015/12/weekly-update-2015-12-16.html

```diff
@@ -0,0 +1,250 @@
+---
+layout: blog
+root: ../../..
+author: Anelda van der Walt
+title: "November 11 - December 16, 2015: 2016 Steering Committee Election, Projects, Instructor Training and More, 
+Lessons, Workshop Feedback, and Data Carpentry is Hiring."
+date: 2015-12-16
+time: "01:30:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p><strong>Highlights</strong></p>
+<ul>
+  <li>
+    The Software Carpentry Foundation will hold its 
+    <a href="{{page.root}}/blog/2015/12/call-for-candidates-elections-2016.html">annual Steering Committee
+    election</a> in February 2016.  
+    Active Software Carpentry community members are invited to both stand for and vote in the elections.
+  </li>
+  <li>
+    The first candidate for the Steering Committee elections is 
+    <a href="{{page.root}}/blog/2015/12/scf-nomination-weaver.html">Belinda Weaver</a>. Learn more about her 
+    history with Software Carpentry and see what she plans to do for the community in 2016.
+  <li>
+    Want to contribute to Software Carpentry, but don't know how? 
+    There's a <a href="{{page.root}}/blog/2015/11/miscellaneous-projects.html">fantastic array of projects</a> 
+    including development of new lessons, fixing up old ones, and much more. 
+    Jump in or contact us if you're interested to participate.
+  </li>
+</ul>
+<strong><p>Other News</strong></p>
+<ul>
+   <li>The past month saw a huge amount of Software Carpentry activities including:
+	<li>Instructor Training events and feedback</li>
+	<li>Many Software Carpentry workshops were ran and several people provided great write-ups about these. 
+	    Write-ups are also available for a number of other workshops from which the Software Carpentry community
+	    could learn a great deal.</li>
+	<li>A few exciting developments in Data Carpentry</li>
+	<li>New lessons were developed and ideas to improve existing lessons were discussed</li>  
+	<li>Several other posts have been published by community members</li>
+  </li>
+</ul> 
+<!-- end excerpt -->
+<p><strong>Instructor Training and Related Info</strong></p>
+<ul>
+  <li>
+    The <a href="{{page.root}}/blog/2015/11/applications-for-december.html">6 week online instructor training</a> 
+    was recently completed. The final step before qualifying as instructors will take place in early 2016 together 
+    with trainees from the 2 day instructor training workshops.
+  </li>
+  <li>
+    Applications for the <a href="{{page.root}}/blog/2015/11/december-2015-team-selection-2015-11-24.html">December
+    instructor training</a> came from 226 individuals from 45 submissions across a wide range of locations.  
+    Only 9 teams were selected to participate.
```

Comment:
We were only able to take 9 of those teams this time around, but hope to work with the others early in 2016.